packages/http-client-java/generator/http-client-generator-test/src/main/java/com/cadl/multicontenttypes/MultipleContentTypesOnRequestAsyncClient.java

```diff
@@ -92,8 +90,6 @@ public Mono<Response<Void>> uploadBytesWithSingleBodyTypeForMultiContentTypesWit
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Mono<Response<Void>> uploadBytesWithMultiBodyTypesForMultiContentTypesWithResponse(String contentType,
         BinaryData data, RequestOptions requestOptions) {
-        // Convenience API is not generated, as operation 'uploadBytesWithMultiBodyTypesForMultiContentTypes' is
```

Comment:
checking on the javadoc and error message when user does not set `convenientAPI(false)`